src/screens/playing.rs

```diff
@@ -18,12 +27,27 @@ pub(super) fn plugin(app: &mut App) {
     );
 }
 
+fn playing_screen(In(id): In<Entity>, mut commands: Commands) {
+    commands
+        .entity(id)
+        .add_fn(widget::ui_root)
+        .insert((Name::new("Playing screen"), StateScoped(Screen::Playing)))
+        .with_children(|_children| {
+            // TODO: Spawn playing screen UI here.
```

Comment:
```suggestion
            // Spawn our playing screen UI like HUDs here.
```

`TODO` sounds like something we forgot to add to the template.

Reply:
Done: https://github.com/TheBevyFlock/bevy_quickstart/pull/231/commits/f891472c0e0eeeeb1ad89d722ca4eb1a7ff76440